cylc/flow/task_pool.py

```diff
@@ -625,14 +625,19 @@ def release_runahead_task(self, itask: TaskProxy) -> None:
         if next_point is not None:
             parent_points = itask.tdef.get_parent_points(next_point)
             n_task = None
-            if (not parent_points or
-                    all(x < self.config.start_point for x in parent_points)):
+            if (
+                (
+                    not parent_points
+                    or all(x < self.config.start_point for x in parent_points)
+                )
+                or
+                (
+                    itask.tdef.get_abs_triggers(next_point)
+                )
+            ):
```

Comment:
(this one already fixed on the integer flow label branch ... looking back after rebase conflicts on that PR!)